cmake/FindThreads.cmake

```diff
@@ -0,0 +1,250 @@
+# - This module determines the thread library of the system.
```

Comment:
Why do you need this file?
We do not use pthreads since we rely on C++11 which brings its own threading.